test/integration/util/util.go

```diff
@@ -102,19 +102,12 @@ func (m *MinishiftRunner) IsCDK() bool {
 	return strings.Contains(cmdOut, "minishift setup-cdk [flags]")
 }
 
-func (m *MinishiftRunner) EnsureRunning() {
-	if m.GetStatus() != "Running" {
-		m.Start()
-	}
-	m.CheckStatus("Running")
-}
-
-func (m *MinishiftRunner) IsRunning() bool {
-	return m.GetStatus() == "Running"
+func (m *MinishiftRunner) IsMinishiftRunning() bool {
+	return strings.Contains(m.GetStatus(), "Minishift:  Running")
```

Comment:
@gbraad @agajdosi Since right now we are only consuming  Minishift status so only add this function in future if we need `IsOpenshiftRunning` then it can be added.

Reply:
+1 for adding `IsOpenshiftRunning`.

Reply:
If also believe, as we already show the status, that `isOpenShiftRunning` should be implemented.

Reply:
ONLY if tests are available for this...

Reply:
@LalatenduMohanty @gbraad isOpenShiftRunning() could have some use, but to apply it to features could take some time, so I would suggest to save @praveenkumar's time and create an issue for this topic and we can add them later. For this PR the isOpenShiftRunning() can be added, however I have no problem with adding them later by myself.